thunder/tests/test_nvfuser_remat.py

```diff
@@ -309,10 +309,12 @@ def test_find_cut_dropout(executor, device, _):
     consumer = nvfuser_symbols[2]
     ext_producer_outputs = find_external_producer_outputs(utils.consumers(trace), (), producer, consumer)
     cut = find_cut(ext_producer_outputs, producer, consumer)
-    # Note t5 is the boolean mask for dropout. It should be chosen over the t6
-    # that is the float32 mask. See this issue: "The Recomputation Algorithm on
-    # Dropout choses a float32 mask to save"
-    assert cut == ("t0", "t5", "t9")
+    assert cut[0] == producer.args[0].name
+    # Note cut[1]/producer.output[0] is the boolean mask for dropout. It should
+    # be chosen over the float32 mask. See this issue: "The Recomputation
+    # Algorithm on Dropout choses a float32 mask to save"
+    assert cut[1] == producer.output[0].name
+    assert cut[2] == producer.output[1].name
```

Comment:
I don't know why CI's trace context is changing the name of the tensors.... I cannot repro this on my local machine, which is very strange.